crates/proto/src/xfer/dnssec_dns_handle.rs

```diff
@@ -229,6 +230,45 @@ where
     }
 }
 
+/// Returns the current system time as Unix timestamp in seconds.
+fn current_time() -> u32 {
```

Comment:
Would like the extraction of this as a separate commit. Would prefer this method to live near the bottom of the module (but before any inline `tests` modules).